### ISSUE TITLE
Tab scrolling to End of Page instead of indenting.

### ISSUE THREAD

**Issue by Panther2804** [alexwforsythe/code-blocks#144]
See this post for an introduction to the Problem:
[Tab scrolling to End of Page instead of indenting.](https://support.google.com/docs/thread/45544838?hl=en
)

I've used code Blocks excessively within this document.

Also: Code Blocks deletes already formatted code when trying to format another piece of code within the same document.

**Comment by catherinetsang** [comment]
I think the issue you are describing is similar to mine.
Essentially, **after** I format a block of text using code blocks, and I want to go back and add a new **indented line**, when you press **tab**, it'll add a new line instead of tabbing the line you're on. 

Screenshot after pressing tab, when I wanted to indent `"persistent": false"`

![image](https://user-images.githubusercontent.com/28896410/82275806-a225ac00-9951-11ea-944a-75e7648e4419.png)

**Comment by Panther2804** [comment]
Yes. This is what I mean. I have found the cause of the issue(on the surface, at least):
When Codeblocks formats code, it puts some (if not all, I'm not sure) parts of your document into a table, in my case, I found that my whole document had been converted into a table.

I fixed this by downloading the Document and converting the Table to text in word.

**Comment by imkharn** [comment]
The following autohotkey script fixes the issue for Windows users by intercepting the tab key, preserving your clipboard, putting tab into your clipboard, pressing control-v, then restoring your clipboard. 
The word "docs" has to be in the active window title, otherwise it wont intercept your tab key, so you should be able to leave this script running at all times. 
Install autohotkey, then put the below text into a text file and rename the file extension to .ahk and double click it.

AutoTrim, Off
SetTitleMatchMode, 2
#IfWinActive Docs
*Tab::
temp = %clipboard%
clipboard = %A_Tab%
send ^v
clipboard = %temp%
return
